src/popup/router/components/Modals/SpendSuccess.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <Modal
+    class="spend-success"
+    close
+    @close="resolve"
+  >
+    <Pending />
+    <div>
+      <TokenAmount
+        :amount="getTxAmountTotal(transaction)"
+        :symbol="getSymbol(transaction)"
+        hide-fiat
+      />
+      {{ $t('pages.send.successalert') }}
```

Comment:
```suggestion
      {{ $t('pages.send.successAlert') }}
```
For now we don't have a strong opinion on that, but it should be either camelCase or kebab-case, as every other variables.